Formula/gator.rb

```diff
@@ -0,0 +1,20 @@
+class Gator < Formula
+  desc "CLI Utility for Open Policy Agent Gatekeeper"
+  homepage "https://open-policy-agent.github.io/gatekeeper/website/docs/gator"
+  url "https://github.com/open-policy-agent/gatekeeper.git",
+    tag:      "v3.9.0",
+    revision: "dfd2dd80734e20d30c00f7e729f03ef5ee9730ed"
+  license "Apache-2.0"
+  head "https://github.com/open-policy-agent/gatekeeper.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "gator"
+    bin.install "bin/gator"
+  end
+
+  test do
+    assert_match "gator is a suite of authorship tools for Gatekeeper", shell_output("#{bin}/gator -h")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula